woocommerce/woocommerce-android
# Remove feature flag to show "Get started" button always

Closes: #7510 


### Description
- Remove feature flag to display "Get started" button always.
- ~~Add the final changes to the login prologue UI~~
- Add unit tests
- Handle edge case where user add the `wordPress` keyword as part of their emails. This will make fetching `username` suggestions provide invalid user names containing `wordPress` which is a blacklited word for usernames. 
- ~~Add release notes~~ (No release notes added as we don't want to advertise this change) 

### Testing instructions
Out of the changes added in this PR the only thing to test is to try to create an account using an email with thisformat: 
`whatever+wordPress@email.com`
The account should be created successfully.
